src/mono/msbuild/android/build/AndroidBuild.targets

```diff
@@ -7,9 +7,8 @@
     <EnableDefaultAssembliesToBundle Condition="'$(EnableDefaultAssembliesToBundle)' == ''">false</EnableDefaultAssembliesToBundle>
   </PropertyGroup>
 
-  <UsingTask Condition="'$(RunAOTCompilation)' == 'true'"
-             TaskName="NdkToolFinderTask"
-             AssemblyFile="$(MobileBuildTasksAssemblyPath)" />
+  <!-- Import default AOT arguments -->
+  <Import Condition="'$(RunAOTCompilation)' == 'true'" Project="$(RepoTasksDir)AotCompilerTask\MonoAOTCompiler.props" />
```

Comment:
The path to the props file needs generically set similarly to how we locate the task assembly. 

See https://github.com/dotnet/runtime/blob/fc55602d39fccddc37272c56a9dadf59492b57a2/src/mono/msbuild/android/build/AndroidBuild.InTree.targets#L3

Reply:
It'll need to be shipped as well, so whatever property you define, add one here too.

https://github.com/dotnet/runtime/blob/fc55602d39fccddc37272c56a9dadf59492b57a2/src/mono/nuget/Microsoft.NET.Runtime.MonoTargets.Sdk/Sdk/MonoTargetsTasks.props.in#L2-L5

Reply:
Thanks, please see: https://github.com/dotnet/runtime/pull/111545/commits/0b338b83c10b2f54a2f8d4d804ac840239d00f3c

Reply:
- Added a functional tests for testing library mode on CI,
- Verified that the `LibraryBuilder.Sdk.10.0.0-dev.nupkg` contains the props file:
```
x _rels/.rels
x Microsoft.NET.Runtime.LibraryBuilder.Sdk.nuspec
x Icon.png
x LICENSE.TXT
x Sdk/AutoImport.props
x Sdk/Sdk.props
x Sdk/AndroidBuild.props
x Sdk/AndroidBuild.targets
x Sdk/AppleBuild.props
x Sdk/AppleBuild.targets
x Sdk/LibraryBuilder.props
x Sdk/LibraryBuilder.targets
x Sdk/CommonMobileBuild.props
x Sdk/MonoAOTCompiler.props
...
```

- Verified that the properties are properly included with E2E test